request/js/main.js

```diff
@@ -0,0 +1,109 @@
+$(function() {
+
+    // QUESTIONS (by default the form starts by asking whether the request is a referral or not)
+    // Will most likely need to be moved to a JSON file at some point in the near future
+    var referralQuestionList = [
```

Comment:
Perhaps add fields for type (radio, dropdown) and options for non-text questions.